File: lib-segment/src/main/java/websiteschema/mpsegment/tools/accurary/OtherErrorAnalyzer.java
package websiteschema.mpsegment.tools.accurary;

import websiteschema.mpsegment.core.WordAtom;

class OtherErrorAnalyzer extends AbstractErrorAnalyzer {

    @Override
    public boolean analysis(WordAtom expect, String possibleErrorWord) {
        increaseOccur();
        addErrorWord(expect.word);
        return true;
    }
}

File: lib-segment-scala/src/main/scala/websiteschema/mpsegment/tools/accurary/OtherErrorAnalyzer.scala
package websiteschema.mpsegment.tools.accurary

import websiteschema.mpsegment.core.Word

class OtherErrorAnalyzer extends AbstractErrorAnalyzer {

  override def analysis(expect: Word, possibleErrorWord: String): Boolean = {
    increaseOccur()
    addErrorWord(expect.name)
//    println(possibleErrorWord + " -- " + expect.word)
    return true
  }
}
